<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Notepad</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="minimumSize">
       <size>
        <width>741</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>741</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>759</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="file_new"/>
    <addaction name="file_open"/>
    <addaction name="file_save"/>
    <addaction name="file_saveas"/>
    <addaction name="file_preview"/>
    <addaction name="file_print"/>
    <addaction name="file_quit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="edit_copy"/>
    <addaction name="edit_cut"/>
    <addaction name="edit_paste"/>
    <addaction name="edit_redo"/>
    <addaction name="edit_undo"/>
    <addaction name="edit_select_all"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="file_new"/>
   <addaction name="file_open"/>
   <addaction name="file_save"/>
   <addaction name="file_print"/>
   <addaction name="edit_copy"/>
   <addaction name="edit_cut"/>
   <addaction name="edit_paste"/>
   <addaction name="edit_undo"/>
   <addaction name="edit_redo"/>
  </widget>
  <action name="file_new">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="file_open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="file_save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="file_saveas">
   <property name="text">
    <string>&amp;Save As</string>
   </property>
  </action>
  <action name="file_print">
   <property name="text">
    <string>&amp;Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="edit_copy">
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="edit_cut">
   <property name="text">
    <string>&amp;Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="edit_paste">
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="edit_redo">
   <property name="text">
    <string>&amp;Redo</string>
   </property>
  </action>
  <action name="edit_undo">
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="edit_select_all">
   <property name="text">
    <string>&amp;Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="file_quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="file_preview">
   <property name="text">
    <string>&amp;Print Preview</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>edit_copy</sender>
   <signal>activated()</signal>
   <receiver>textEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit_cut</sender>
   <signal>activated()</signal>
   <receiver>textEdit</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit_paste</sender>
   <signal>activated()</signal>
   <receiver>textEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit_redo</sender>
   <signal>activated()</signal>
   <receiver>textEdit</receiver>
   <slot>redo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit_undo</sender>
   <signal>activated()</signal>
   <receiver>textEdit</receiver>
   <slot>undo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit_select_all</sender>
   <signal>activated()</signal>
   <receiver>textEdit</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>file_quit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
